react/the_react_ecosystem/project_shopping_cart.md

```diff
@@ -37,6 +37,8 @@ By now you've come far from your React-baby days, with tools like routers and te
         }
         ~~~
 
-    You can learn more about how these work at the [Netlify](https://docs.netlify.com/routing/redirects/) and [Vercel](https://vercel.com/docs/frameworks/vite#using-vite-to-make-spas) docs respectively.
+      3. **Cloudflare Pages**: As of the time of writing, unlike Netlify and Vercel, no additional steps are required as the default behaviour will correctly handle single page applications.
+
+    You can learn more about how these work at the  [Netlify](https://docs.netlify.com/routing/redirects/), [Vercel](https://vercel.com/docs/frameworks/vite#using-vite-to-make-spas) and [Cloudflare](https://developers.cloudflare.com/pages/platform/serving-pages/), docs respectively.
```

Comment:
```suggestion
    You can learn more about how these work at the [Netlify](https://docs.netlify.com/routing/redirects/), [Vercel](https://vercel.com/docs/frameworks/vite#using-vite-to-make-spas) and [Cloudflare](https://developers.cloudflare.com/pages/platform/serving-pages/) documentation respectively.
```

Reply:
removed an extra space and I think the last comma isn't needed. More descriptive links would be even better if you can come up with something :)

Thanks for adding these documentation links!